actors/common/src/main/java/org/sunbird/learner/actors/SchedularActor.java

```diff
@@ -69,7 +70,8 @@ private void schedule(Request request) {
             "calling bulkUploadBackGroundJobActor for processId from schedular actor "
```

Comment:
Change log statement as:

"SchedularActor: scheduleBulkUpload called with processId " + {0} + " and type " + {1}